### ISSUE TITLE
If logged in with/as a galaxy, and stars are locked up, "spawn" should be replaced with "lockup/release" link

### ISSUE THREAD

**Issue by Fang-** [urbit/bridge#400]
People generally expect to be able to "spawn" their stars, even if they were in lockup, so linking to the release contract page alongside/instead of the spawn action page would help them find the right way to get their stars.

**Comment by Fang-** [comment]
Also it incorrectly suggests locked up stars as spawn candidates, making this double confusing. See also urbit/support#18.

**Comment by Fang-** [comment]
Interesting to note that urbit/support#18 also suggests that "issue point" was clickable even though all the galaxy's stars had already been spawned.

**Comment by kingdonb** [comment]
I noticed this myself this weekend. (First star from ~del is spawned now, ~fipdel!)

If I understand correctly, the stars are released in order and there is no control for the galaxy to decide which stars are released in which order. (I can provision any of the ~59 released stars if I allocate them all to an ethereum address, but I have to allocate them in order from the released star pool)

**Comment by Fang-** [comment]
> If I understand correctly, the stars are released in order

@kingdonb This is correct. The release contracts hold a "stack" of stars, and you can only withdraw the top one. This behavior cannot be changed.

**Comment by kingdonb** [comment]
Cool, then it would definitely be beneficial for bridge to reflect this, since it is suggesting star names that cannot possibly be used. (But not critical, as I was able to figure out the issue from errors I got, and move on.)

Thanks!
